HerPublicWebsite/Controllers/ReferralRequestFollowUpController.cs

```diff
@@ -0,0 +1,58 @@
+using System.Threading.Tasks;
+using HerPublicWebsite.Models.ReferralRequestFollowUp;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using HerPublicWebsite.BusinessLogic.Services.ReferralFollowUps;
+using HerPublicWebsite.Models.Enums;
+using HerPublicWebsite.BusinessLogic.Models;
+
+namespace HerPublicWebsite.Controllers;
+
+[Route("referral-follow-up")]
+public class ReferralRequestFollowUpController : Controller
+{
+    private readonly IReferralFollowUpService referralFollowUpService;
+
+    public ReferralRequestFollowUpController(
+        IReferralFollowUpService referralFollowUpService
+    )
+    {
+        this.referralFollowUpService = referralFollowUpService;
+    }
+
+    [HttpGet("already-responded")]
+    public IActionResult AlreadyResponded()
+    {
+         return View("ReferralRequestFollowUpAlreadyResponded");
+    }
+
+    [HttpGet("respond-page/{token}")]
```

Comment:
We could probably get rid of the `respond-page` here. I also wonder if we should pass the token as a query string rather than a URL parameter, so that:
- `GET /referral-follow-up?token=abcd1234` = main follow up page
- `POST /referral-follow-up` = submit follow up
- `GET /referral-follow-up/response-recorded` = confirmation
- `GET /referral-follow-up/already-responded` = already responded